logging/testutils.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Google LLC
```

Comment:
This should be in a test file so it is not added to the public api of this package.